src/components/ui/ProgressBar/styles.ts

```diff
@@ -0,0 +1,26 @@
+import { StyleSheet } from "react-native";
+import { montserrat } from "@app/utils";
+import { RFValue } from "react-native-responsive-fontsize";
+
+interface StyleProps {
+  labelColor: string;
+}
+
+export const useStyles = ({ labelColor }: StyleProps) =>
+  StyleSheet.create({
+    container: {
+      paddingHorizontal: 10,
+      width: "100%",
+      marginVertical: 15,
+    },
+    labelText: {
+      marginBottom: 10,
```

Comment:
Think it would be good to add this to the grid object in utils so we can globally change our padding and margins consistently.